### ISSUE TITLE
[ALM Accelerator - Feature]: Provide a configuration-based approach to ensure environment settings are set

### ISSUE THREAD

**Issue by devkeydet** [microsoft/coe-starter-kit#2084]
### Describe the solution you'd like

Often, in order for an app to be in a working state, environment settings need to be set.  

![image](https://user-images.githubusercontent.com/4499296/154568229-6343ee6b-4205-4242-877b-492da6470da0.png)

These values can be set using Dataverse API using the Organization table.
https://docs.microsoft.com/en-us/powerapps/developer/data-platform/reference/entities/organization

Expand the customDeploymentSettings.json concept to read these values and apply them.

AB#1258

**Comment by mikefactorial** [comment]
Agree. We should expand the deployment configuration to be a preflight check that validates these types of things along with all of the other dependencies required to deploy a solution (e.g. does this solution rely on Dynamics, does this solution rely on other base solutions etc.).

**Comment by AmarSaiRam** [comment]
According to https://learn.microsoft.com/en-us/power-platform/alm/pipelines, Power Platform Pipelines are now the recommended approach for Application Lifecycle Management (ALM). The ALM Accelerator is no longer actively maintained or supported. Therefore, this issue is being closed accordingly.
